deployments/helm/nvidia-dra-driver-gpu/templates/clusterrole.yaml

```diff
@@ -2,8 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "nvidia-dra-driver-gpu.name" . }}-role
-  namespace: {{ include "nvidia-dra-driver-gpu.namespace" . }}
+  name: {{ include "nvidia-dra-driver-gpu.name" . }}-clusterrole
 rules:
 - apiGroups: ["resource.nvidia.com"]
```

Comment:
I think more of the rules below could be tightened. for example we don't need any operations on `deviceClasses` anymore, and we never Create or delete `ComputeDomains` oursevles (only the user does).